Integrations/VMRay/VMRay.py

```diff
@@ -490,6 +490,56 @@ def get_submission(submission_id):
     response = http_request('GET', url_suffix=suffix)
     return response
 
+def check_file_command():
+    file_hash_input = demisto.args().get('file')
+    file_hashes = file_hash_input.split(',')
+    for file_hash in file_hashes:
+        if len(file_hash) != 32 and len(file_hash) != 40 and len(file_hash) != 64:
+            return_error('Invalid input hash')
+        hash_type = 'md5'
+        if len(file_hash) == 40:
+            hash_type = 'sha1'
+        if len(file_hash) == 64:
+            hash_type = 'sha256'
+
+        raw_response = check_file(file_hash, hash_type)
+        data = raw_response.get('data')
+
+        if len(data) == 0:
+            return_outputs(
+                'No submission found in VMRay for hash : {}'.format(file_hash),
```

Comment:
we should return dbot score entry here as well, with `Unknown` verdict

Reply:
As this is no longer providing the 'file' functionality, I don't think it should return any DBot Score. The lookup function is to find out whether a submission has already occurred for the sample, not to look up it's value. The lookup for the sample can then be acheived using the returned samplID from the lookup function.